examples/specs/normalized/trellis_row_column_normalized.vl.json

```diff
@@ -23,6 +23,14 @@
       "y": {
         "field": "Miles_per_Gallon",
         "type": "quantitative"
+      },
+      "color": {
+        "field": "Cylinders",
+        "type": "nominal"
+      },
+      "shape": {
```

Comment:
Why over-encoding both shape and color?  (Over-encoding is generally ineffective.)